install.sh

```diff
@@ -58,6 +58,7 @@ echo "==== Create 'dgfem' conda environment"
 
 conda create --name dgfem --yes
 
+source $MY_CONDA_DIR/bin/activate
```

Comment:
This seems redundant with the command just below. It's not obvious why it's needed or why it helps. At the least, it needs a comment justifying its existence.

Reply:
The reason is (on quartz):

```bash
$ source /g/g91/diener3/miniforge3/bin/activate dgfem
$ which conda
/usr/bin/conda
$ source /g/g91/diener3/miniforge3/bin/activate
$ which conda
~/miniforge3/bin/conda
```

I'll attempt to put this in a comment.

Reply:
Wow, that's ridiculous, and (IMO) a pretty severe upstream (conda) bug. I suspect this will continue to bite us hard if we don't get it fixed upstream. Could you report that and add the bug link to your comment? https://github.com/conda/conda

Reply:
I'm just happy that (in this case) it wasn't a shameful user and usage error.  This issue was driving me insane on quartz.

Reply:
I'd really like a link to an upstream bug here.

Reply:
Added a link to https://github.com/conda/conda/issues/10133

Reply:
Thanks!